config/datatypes_conf.xml.sample

```diff
@@ -333,6 +336,8 @@
     <sniffer type="galaxy.datatypes.images:Xpm"/>
     <sniffer type="galaxy.datatypes.images:Eps"/>
     <sniffer type="galaxy.datatypes.images:Rast"/>
+    <sniffer type="galaxy.datatypes.msa:Stockholm_1_0" />
```

Comment:
Someone with more experience should comment here, but I guess nothing should come after the image datatypes.